modules/ethereumNode.js

```diff
@@ -488,6 +488,22 @@ class EthereumNode extends EventEmitter {
 
                 const proc = spawn(binPath, args);
 
+                /*
+                    Assume startup succeeded after 5 seconds. At this point 
+                    IPC connections are usually possible.
+                */
+                setTimeout(() => {
```

Comment:
This should only be started, in `.stderr.on('data', ...)`, at least then we know for sure that it at least started